python/lsst/ts/logging_and_reporting/jira.py

```diff
@@ -0,0 +1,171 @@
+        else:
+            raise Exception(
+                f"Error getting user timezone from {os.environ.get('JIRA_API_HOSTNAME')}: "
+                f"{response.status_code} - {response.text}"
+            )
+
+        # convert the utc times to user timezone
+        start_dayobs_user = start_dayobs_utc.astimezone(user_timezone)
+        end_dayobs_user = end_dayobs_utc.astimezone(user_timezone)
+
+        start_dayobs_str = start_dayobs_user.strftime("%Y-%m-%d %H:%M")
+        end_dayobs_str = end_dayobs_user.strftime("%Y-%m-%d %H:%M")
+
+        jql_query = (
+            f'project = OBS AND (created >= "{start_dayobs_str}" '
+            f'AND created < "{end_dayobs_str}")'
+        )
+        fields = f"key,summary,updated,created,status,system,{OBS_SYSTEMS_FIELD}"
+        url = f"https://{os.environ.get('JIRA_API_HOSTNAME')}/rest/api/latest/search/jql?jql={quote(jql_query)}&fields={fields}"
+        response = requests.get(url, headers=headers)
+
+        if response.status_code == 200:
+            issues = response.json()["issues"]
+            return [
+                {
+                    "key": issue["key"],
+                    "summary": issue["fields"]["summary"],
+                    "updated": issue["fields"]["updated"],
+                    "created": issue["fields"]["created"].split(".")[0],
+                    "status": issue["fields"]["status"]["name"],
+                    "system": get_system_names(issue["fields"][OBS_SYSTEMS_FIELD]),
+                }
+                for issue in issues
+            ]
+        raise Exception(
+            f"Error getting issues from {os.environ.get('JIRA_API_HOSTNAME')}: "
+            f"{response.status_code} - {response.text}"
+        )
+
+    @staticmethod
+    def parse_obs_issues_array_to_plain_text(obs_issues):
+        """Parse the OBS issues array to plain text.
+
+        Parameters
+        ----------
+        obs_issues : `list`
+            List of OBS issues
+
+        Notes
+        -----
+        Each list entry (issue) of obs_issues must be dictionary
+        with the following keys:
+        - key: The key of the issue
+        - status: The status of the issue
+        - summary: The summary of the issue
+        - created: The creation datetime of the issue
+        - updated: The last updated datetime of the issue
+        - system: The system "customfield_10476" of the issue
+
+        If a key is missing, it will be replaced by a "None".
+
+        Returns
+        -------
+        str
+            The OBS issues in plain text format
+        """
+        lines = []
+        for issue in obs_issues:
+            lines.append(
+                f"{issue.get('key')} - {issue.get('status')} - {issue.get('summary')} "
+                f"Created: {issue.get('created')}, Last Updated: {issue.get('updated')}, "
+                f"Impacting: {issue.get('system')}"
+            )
+        return "\n".join(lines)
```

Comment:
On the LOVE-manager this is used to arrange a plain text format of the list of issue to be included on the email for the night summary, however for ts_logging we will have custom formating so I think we are not going to use this method.